kytos/utils/napps.py

```diff
@@ -248,6 +249,18 @@ def match(napp):
         napps = NAppsClient().get_napps()
         return [napp for napp in napps if match(napp)]
 
+    def install_requirements(self):
+        """Install NApp requirements."""
+        folder = self._installed / self.user / self.napp
+        os.chdir(folder)
+        requirements_file = "requirements/dev.txt"
+        if os.path.exists(requirements_file):
+            LOG.info('Installing dependencies:')
+            subprocess.call([sys.executable, "-m", "pip", "install", "-r",
+                             requirements_file])
```

Comment:
This will work for now, as Kytos Utils still runs on the same machine as Kytos, but the ideal way would be to run this directly on the server side, to avoid adding more dependency to the current scenario.

Reply:
Cool @hdiogenes, maybe we can move these modifications to server side later. Thanks a lot!